assets/src/edit-story/components/panels/textStyle/color.js

```diff
@@ -33,9 +33,15 @@ import { BACKGROUND_TEXT_MODE } from '../../../constants';
 import { FillNone, FillFilled, FillHighlighted } from '../../../icons/';
 import { Color, Label, Row, ToggleButton } from '../../form';
 import { useKeyDownEffect } from '../../keyboard';
-import { useCommonColorValue, getCommonValue } from '../utils';
+import {
+  useCommonColorValue,
+  getCommonValue,
+  useCommonObjectValue,
+} from '../utils';
 import getColorPickerActions from '../utils/getColorPickerActions';
+import { metricsForNewPadding } from '../utils/metricsForNewPadding';
```

Comment:
^ Could we rename this to something more accurate/expressive, e.g. `metricsForTextPadding` (in case that's correct) or something similar? It's hard to understand what is meant by `newPadding`. WDYT?

Reply:
Great idea. `new` I guess as in new values that are calculated against the old ones.